editor/dependency_editor.cpp

```diff
@@ -766,6 +766,8 @@ bool OrphanResourcesDialog::_fill_owners(EditorFileSystemDirectory *efsd, HashMa
 	}
 
 	bool has_children = false;
+	String main_scene = GLOBAL_GET("application/run/main_scene");
+	String scene_name = main_scene.substr(main_scene.find("//") + 2);
```

Comment:
This will only work if your main scene is immediately in `res://`, I'd suggest just:
```suggestion
	String scene_name = main_scene.get_file();
```
If the path is, for example, `res://scenes/my_scene.tscn` the existing code will return `scenes/my_scene.tscn` which won't match `efsd->get_file(i)` right?